spring-boot/admin/src/main/resources/templates/admin/fragments/api-keys/list.html

```diff
@@ -237,5 +175,15 @@ <h2 th:text="#{admin.api-keys.subtitle}">[Post Administration</h2>
         </div>
         <div th:include="admin/fragments/scripts :: scripts" th:remove="tag">Javascript Libraries</div>
         <div th:include="admin/fragments/scripts :: initializeDataTables" th:remove="tag">Javascript Libraries</div>
+        <div th:include="admin/fragments/scripts :: apiKeyRemoveDialog" th:remove="tag">Javascript Libraries</div>
+
+    <script>
+        function createInfoBoxMessageAndDeleteKey(apiKeyId, consumerName){
```

Comment:
name could be improved, see other comment related to this method name

Reply:
Done.